breadth_first_search/BreadthFirstSearch.py

```diff
@@ -0,0 +1,31 @@
+import collections
+
+def bfs(graph, root):
+    returnprint = 'BFS Order: '
+    seen, queue = set([root]), collections.deque([root])
+    while queue:
+        vertex = queue.popleft()
+        returnprint += str(vertex) + '  '
+        for node in graph[vertex]:
+            if node not in seen:
+                seen.add(node)
+                queue.append(node)
+    print(returnprint)
+
+def visit(n):
+    print(n)
+
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpisevponc/breadth_first_search/BreadthFirstSearch.py
+++ b/tmp/tmpisevponc/breadth_first_search/BreadthFirstSearch.py
@@ -15,9 +15,10 @@
 def visit(n):
     print(n)
 
+
 if __name__ == '__main__':
-    graph = {0: [1, 2], 1: [2, 0], 2: []} 
-    bfs(graph, 1) # 1 2 0
+    graph = {0: [1, 2], 1: [2, 0], 2: []}
+    bfs(graph, 1)  # 1 2 0
 
     graph2 = {0: [1, 3], 1: [3, 2], 2: [2, 1], 3: []}
     bfs(graph2, 2) # 2 1 3
```